src/server/api/boards/boards.js

```diff
@@ -100,6 +100,47 @@ function addCardToColumn(board, columnId, cardName) {
   });
 }
 
+function validateColumnId(board, columnId, newColumnId) {
+  let oldFound = false;
+  let newFound = false;
+
+  board.columns.forEach(function(element) {
+    if (element._id.toString() === columnId.toString()) {
```

Comment:
`element._id.toString()` is also at line 111. you could put it into a variable. same with the `columnId.toString()` you can put it outside of the `forEach` since it will never change during the iteration